server/functions/timestamptz.go

```diff
@@ -50,6 +50,9 @@ var timestamptz_in = framework.Function3{
 		//if b.Precision == -1 {
 		//	p = b.Precision
 		//}
+		if input == "infinity" || input == "-infinity" {
+			return input, nil // TODO: might need different representation in time.Time value
+		}
```

Comment:
What about for `"Infinity"`, `"INFINITY"`, etc.? Also, we must not have any tests for this, as we're returning two different types in this function. Here we're returning a string, which is invalid according to the return type (`pgtypes.TimestampTZ`).